app/views/request/show.html.erb

```diff
@@ -11,6 +11,10 @@
 <% if flash[:request_sent] %>
   <%= render :partial => 'request_sent',
              :locals => { :info_request => @info_request } %>
+  <% content_for :analytics_events do
+    raw track_analytics_event(AnalyticsEvent::Category::REQUEST_ACTION,
```

Comment:
It would be good to be know which request the event belongs to so we could capture in as an event label with something like:

```ruby
raw track_analytics_event(AnalyticsEvent::Category::REQUEST_ACTION,
                          AnalyticsEvent::Action::NEW_REQUEST,
                          :label => @info_request.url_title)
```